src/morph_server/src/Respond.re

```diff
@@ -47,5 +49,6 @@ let respond =
         |> Lwt.return
       });
     read_stream();
+  | _ => raise(Incorrect_Body)
```

Comment:
Should it raise? Shouldn't it respond with something like `400`?

Reply:
Since this is a type error I think you might want to just raise and catch it at development time. It might make sense to respond with 500 and "Unsupported body type" instead. What do you think?

Reply:
I don't think it should raise... We can add an issue for it though. But I believe we should give developers a way to control this in a more transparent way.

Reply:
I'll see what I can do with this, I think I'm not handling errors correctly on httpaf anyway.

Reply:
👍 If we go the multi body type rabbit hole handling this correctly is critical.

Reply:
I kept the raise here but handle it before we get to this point